packages/cardano-services/src/Asset/types.ts

```diff
@@ -16,3 +16,15 @@ export interface NftMetadataService {
    */
   getNftMetadata(asset: AssetPolicyIdAndName): Promise<Asset.NftMetadata | undefined>;
 }
+
+/**
+ * Service to get CIP-35 token metadata for a given subject
```

Comment:
```suggestion
 * Service to get CIP-38? token metadata for a given subject
```
I've just noticed 35 was claimed by another CIP first, so this one has tentatively been named 38